### PR TITLE
Update proofpoint-pod after changing in the connector

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
Hello @squioc , I add the `event.action` to `msgParts`. But can't see `disposition` field in the `msgPartURL` samples.

Parent Task: ProofPoint PoD: forward disposition in sub-events [43633](https://github.com/SekoiaLab/platform/issues/43633)

### CODE REVIEW

**Comment by Adamowoc** [comment]
Could you link the related proofpoint-pod connector PR please ?

File: ProofPoint/proofpoint-pod/tests/test_msgparts.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
         "email"
       ],
       "dataset": "msgParts",
+      "action": "inline",
```
Comment (squioc) [inline ProofPoint/proofpoint-pod/tests/test_msgparts.json:19]: The JSON document in the field `message` should be updated too with the new property `disposition`

File: ProofPoint/proofpoint-pod/ingest/parser.yml [hunk 2]
```diff
@@ -95,6 +95,7 @@ stages:
       - set:
           event.category: ["email"]
           event.dataset: "msgParts"
+          event.action: "{{json_event.message.msgParts.disposition}}"
```
Comment (squioc) [inline ProofPoint/proofpoint-pod/ingest/parser.yml:98]: The disposition field is at the root of the event
```suggestion
          event.action: "{{json_event.message.disposition}}"
```

**Review by squioc — CHANGES_REQUESTED** [review]
`event.action` is missing from the stage `set_fields_for_message_parts_url`.
You should also update the test `test_msgparts_url.json` to add the disposition in the orignal message.

File: ProofPoint/proofpoint-pod/tests/test_msgparts.json (hunk 1 above)
Comment (TOUFIKIzakarya) [reply ProofPoint/proofpoint-pod/tests/test_msgparts.json:19]: Ok i have understand, thank you 🙏.

**Review by squioc — APPROVED** [review]

**Pull request merged**